modules/administration/pages/ssl-certs.adoc

```diff
@@ -294,3 +310,47 @@ To complete the process, you need to update the database with this command:
 ----
 
 If you use bootstrap, remember to also update your bootstrap scripts to reflect the new certificate information.
+
+
+
+== Replace Certificates with a Third Party Certificiate
+
+You can replace active certificates on your {productname} installation with a new third party certificate.
+To replace the certificates, you can replace the installed CA certificate RPM with a new RPM containing the third party certificate, and then update the database.
+
+This procedure is similar to the one described in "Create and Replace CA and Server Certificates" with the difference, that we import the certificates generated by an external PKI.
+
+
+.Procedure: Replacing Existing Certificates
+
+. On the {productname} Server, at the command prompt, move the old certificate directory to a backup location:
++
+----
+mv /root/ssl-build /root/old-ssl-build
+----
+
+. Generate a CA certificate RPM from the new certificate:
++
+----
+rhn-ssl-tool --gen-ca --rpm-only --dir="/root/ssl-build" --from-ca-cert=<Path_to_CA_Certificate>
+----
+
+. Generate a new server certificate RPM:
++
+----
+rhn-ssl-tool --gen-server --rpm-only --dir="/root/ssl-build" --from-server-key=<Server_Key_File> --from-server-cert=<Server_Cert_File>
+----
++
+This command might end with a warning that the server certificate request file could not be found.
+This file is not required, but you can provide it, by copying it into the server directory as `server.csr`.
++
+----
+cp <Certificate_Request_File>.csr /root/ssl-build/<Server_Name>/server.csr
+----
++
+When you repeat the command to generate the server certificate RPM, the warning should not be shown anymore.
+
+Now you have an ssl-build directory similar to the one generated by "Procedure: Creating New Certificates" from the chapter before.
+You can now continue with the "Procedure: Create Combined Certificate RPMs" from the "Create and Replace CA and Server Certificates" chapter.
```

Comment:
@Loquacity @keichwa you are the asciidoc experts. I think there should be links to the sections. But I have no idea how to do it.